tests/component/test.ajax.js

```diff
@@ -16,23 +16,14 @@ describe('test.ajax.js', function () {
       }
     });
     server.listen(6000, function () {
-      PouchDB.ajax({
-        method: 'GET',
-        url: 'http://127.0.0.1:6000/install-cookie',
-        timeout: 10
-      }, function (err, res) {
-        should.equal(res.ok, true, "Server not responding");
-
-        PouchDB.ajax({
-          method: 'GET',
-          url: 'http://127.0.0.1:6000/check-cookie',
-          timeout: 10
-        }, function (err, res) {
-          server.close();
-
-          should.equal(res.ok, true, "Cookie not set");
-          done();
-        });
+      PouchDB.fetch('http://127.0.0.1:6000/install-cookie').then(function () {
```

Comment:
``should.equal(res.ok, true, "Server not responding");`` is missing here. Not sure if important.

Reply:
Those status checks are a bit more useful for callbacks to ensure you error out early, but for promises we would error out here if the server doesnt respond so happy to skip it

Reply:
Ok, just checking. :)